README.md

```diff
@@ -40,23 +47,33 @@ We refer to our [official benchmarks](https://www.kymat.io/userguide.html#benchm
 
 ### How to cite
 
-If you use this package, please cite the following paper:
+If you use Kymatio in a scientific publication, please cite [arXiv preprint 1812.11214](https://arxiv.org/abs/1812.11214).
+Here is the BibTeX reference:
 
-Andreux M., Angles T., Exarchakis G., Leonarduzzi R., Rochette G., Thiry L., Zarka J., Mallat S., Andén J., Belilovsky E., Bruna J., Lostanlen V., Hirn M. J., Oyallon E., Zhang S., Cella C., Eickenberg M. (2019). Kymatio: Scattering Transforms in Python. arXiv preprint arXiv:1812.11214. [(paper)](https://arxiv.org/abs/1812.11214)
+```
```

Comment:
Can't we just link to the BiBTeX reference? This is a bit ugly to put up front (and marginally more useful).

Reply:
I did this so that citing us is really minimal effort: it's a copy-paste from the README (where we have most visibility). I'm OK with making it a link, but that's one click more for whoever is citing us

Reply:
If someone wants to cite us, I don't think they'll be discouraged by clicking a link.